channels/rdpecam/client/CMakeLists.txt

```diff
@@ -17,16 +17,12 @@
 
 define_channel_client("rdpecam")
 
-# swscale is required and is either part of FFMPEG or standalone library
-if(NOT WITH_FFMPEG)
-	find_package(SWScale REQUIRED)
-endif()
-
-if(NOT WITH_OPENH264 AND NOT WITH_FFMPEG)
-	message(FATAL_ERROR "WITH_OPENH264 or WITH_FFMPEG required for CHANNEL_RDPECAM_CLIENT")
+if(NOT WITH_SWSCALE OR NOT WITH_FFMPEG)
+    message(FATAL_ERROR "WITH_FFMPEG and WITH_SWSCALE required for CHANNEL_RDPECAM_CLIENT")
 endif()
 
```

Comment:
@akallabeth: why only FFMPEG supported currently? I've been testing with OpenH264 and it gives better results (better CPU utilization)?

Reply:
@oleg0421 because you use a few functions from `FFMPEG` to manipulate the buffers (not the `swscale` stuff, just the preparation)